chain/network/src/peer_manager/connection/mod.rs

```diff
@@ -328,9 +343,6 @@ impl Pool {
 
     pub fn start_outbound(&self, peer_id: PeerId) -> Result<OutboundHandshakePermit, PoolError> {
```

Comment:
Add a comment (and mention that we do allow loop-back connectoins now)

Reply:
done